packages/flutter/lib/src/material/i18n/time_picker_localizations.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Keep this file in sync with dateTimePatternMap() from package:intl.
+
+/// Maps languages and locales to the respective pattern used to display time of
+/// day.
+///
+/// ## Usage
+///
+/// Given a [Locale], first try looking up a pattern by full locale (e.g.
+/// [Locale.toString]). If not listed, fallback to [Local.languageCode].
+const Map<String, String> timeOfDayLocalizations = const <String, String>{
```

Comment:
It seems like it would be more consistent with the other localisations to have these strings be in the `.arb` files and then handle them just like any other string.

This has the advantage that it avoids any risk that we'll have an `.arb` file for a localization that we don't have in this list.

Reply:
I think having it in the .arb files would be fine, personally, but even if it's not in the .arb files it should definitely be in the localization objects. The point is that this table shouldn't exist, just like in the typography patch.

Reply:
Moved to the .arb files. Also, changed `MaterialLocalizations` to fallback to language code if locale-specific value is missing, then to `en` if that's missing too.